### PR TITLE
Defunct processes

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20702

## Description
Implemented a function that closes the driver & reaps all the child process that are defunct.

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Code Review

## Dependencies
- [x] Phishing v2 Test - Inline
- [x] RasterizeImageTest
- [x] process_email_-_generic_-_test
- [ ] URL Enrichment - Generic v2 - Test
- [x] Rasterize Test
- [x] Impossible Traveler - Test
- [x] Process Email - Generic for Rasterize
- [x] Phishing v2 Test - Attachment
- [x] EWS Mail Sender Test 2
- [ ] Phishing - Core - Test

## Technical writer review
- [ ] [CHANGELOG](https://github.com/demisto/content/blob/9cd4c8ac0ee83bbca22ca91dcd19ec2ab78a4ed1/Integrations/rasterize/CHANGELOG.md)

### CODE REVIEW

File: Integrations/rasterize/rasterize.py [hunk 1]
```diff
@@ -68,6 +68,24 @@ def init_driver(offline_mode=False):
     return driver
 
 
+def quit_driver_and_reap_children(driver):
```
Comment (Itay4) [inline Integrations/rasterize/rasterize.py]: add type hint

**Comment by kirbles19** [comment]
@guyfreund Done.

**Comment by chris4490** [comment]
Is this going to go into a 5.0 Patch? I just came across a customer with
20,858 defunct processes. I know they release resources and shouldn't cause
issues aside from 32-bit machines, but curious if we'll put it into a 5.0
patch.

On Thu, Dec 19, 2019 at 8:09 AM Anar Azadaliyev <notifications@github.com>
wrote:

> Merged #5178
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_demisto_content_pull_5178&d=DwMCaQ&c=V9IgWpI5PvzTw83UyHGVSoW3Uc1MFWe5J8PTfkrzVSo&r=tlLSArGXt0Xt_SwdwWjMOf8U3ag6jIrUDst2VIG1Uu0&m=MSmDAqeLsN1imoFJM5rT1-EBIio2Gna6478AyA9O5kk&s=Zb9KLAoaDBDYy2YDKwRns5WTK1eBZ4Lm_mYz_2ylG7Y&e=>
> into master.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_demisto_content_pull_5178-3Femail-5Fsource-3Dnotifications-26email-5Ftoken-3DAKGMO5P5SRLAN5CYFAAZTLTQZN6DJA5CNFSM4J3BALAKYY3PNVWWK3TUL52HS4DFWZEXG43VMVCXMZLOORHG65DJMZUWGYLUNFXW5KTDN5WW2ZLOORPWSZGOVSZBEEY-23event-2D2897351187&d=DwMCaQ&c=V9IgWpI5PvzTw83UyHGVSoW3Uc1MFWe5J8PTfkrzVSo&r=tlLSArGXt0Xt_SwdwWjMOf8U3ag6jIrUDst2VIG1Uu0&m=MSmDAqeLsN1imoFJM5rT1-EBIio2Gna6478AyA9O5kk&s=fsC5_pekDPrVmOw70X4i9zTZcdCql0FilY8qhKt5xIY&e=>,
> or unsubscribe
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AKGMO5MX6WRMLJ2PZCXSOILQZN6DJANCNFSM4J3BALAA&d=DwMCaQ&c=V9IgWpI5PvzTw83UyHGVSoW3Uc1MFWe5J8PTfkrzVSo&r=tlLSArGXt0Xt_SwdwWjMOf8U3ag6jIrUDst2VIG1Uu0&m=MSmDAqeLsN1imoFJM5rT1-EBIio2Gna6478AyA9O5kk&s=ZR73eeUnNbjjvP0mIJy1-mUa0AKBmhfh7aKevRo87vQ&e=>
> .
>


-- 
Regards,
Chris Carper
Customer Support Engineer
ccarper@paloaltonetworks.com

**Comment by guyfreund** [comment]
Yes, will be released in the next content release.

**Comment by chris4490** [comment]
Awesome. Thank you!

**Pull request merged**